### DISCUSSION TITLE
milvus-querynode  expansion

### DISCUSSION THREAD

**Discussion by Kellen-X** [milvus-io/milvus#37512] [Q&A and General discussion]
I deployed a cluster using kubectl, and the documentation shows that helm is required for scaling, and my current status does not support helm, how can I use kubectl to scale out

**Comment by xiaofan-luan** [comment]
if query qps is not enough, change your querynode replica to add more. 
if index or data can not catch up, change their replica could work.

We still recommend to use helm or K8s operator since it's easier for you to manage

**Reply by Kellen-X** [reply]
I can do this with the K8s operator, but I don't know how to extend the querynode component, I need to extend the querynode component, or the number of pods of the querynode

**Reply by LoveEachDay** [reply]
@Kellen-X when using helm chart, you can use `kubectl scale deploy <milvus-querynode-deploy> --replicas=<expected replicas>` to manually scale out.

Or you can use `helm upgrade` to change querynode replicas. Read more from https://milvus.io/docs/scaleout.md

**Reply by Kellen-X** [reply]
No, I can't install and upgrade using helm, and I can't complete the expansion using a yaml file through kubectl?
kubectl apply -f https://raw.githubusercontent.com/zilliztech/milvus-operator/main/config/samples/milvus_cluster_default.yaml

**Reply by Kellen-X** [reply]
> @Kellen-X当使用helm chart时，可以使用`kubectl scale deploy <milvus-querynode-deploy> --replicas=<expected replicas>`来手动扩容。
> 
> 或者您可以使用`helm upgrade`更改查询节点副本。更多信息请阅读https://milvus.io/docs/scaleout.md

No, I can't install and upgrade using helm, and I can't complete the expansion using a yaml file through kubectl?
kubectl apply -f https://raw.githubusercontent.com/zilliztech/milvus-operator/main/config/samples/milvus_cluster_default.yaml
